src/Compilers/VisualBasic/Portable/Utilities/FeatureUtils.vb

```diff
@@ -0,0 +1,53 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Option Compare Binary
+Option Strict On
```

Comment:
`Option` statements are not necessary. Options are set by the build, and these are the defaults.